libs/httpext/src/lib/http-ext.spec.ts

```diff
@@ -95,43 +95,23 @@ describe('HttpExt', () => {
         )
     });
 
-    /* The first plugin should match the condition and handle the request */
-    expect(pluginA.condition.mock.calls[0][0].request).toEqual({
-      url: 'https://answer-to-the-ultimate-question-of-life.com/',
-      method: 'GET',
-      body: null,
-      headers: {},
-      params: {}
-    });
-    expect(pluginA.condition.mock.results[0].value).toBeTruthy();
-    expect(pluginA.handle.mock.calls.length).toBe(1);
-    expect(pluginA.handle.mock.calls[0][0].request).toEqual({
-      url: 'https://answer-to-the-ultimate-question-of-life.com/',
-      method: 'GET',
-      body: null,
-      headers: {},
-      params: {}
-    });
+    /* The first plugin should match the condition and handle the request. */
+    expect(pluginA.handle).toHaveBeenCalledTimes(1);
 
-    /* The second plugin shouldn't match the condition and never handle */
-    expect(pluginB.condition.mock.calls[0][0].request).toEqual({
-      url: 'https://answer-to-the-ultimate-question-of-life.com/',
-      method: 'GET',
-      body: null,
-      headers: {},
-      params: {}
-    });
-    expect(pluginB.condition.mock.results[0].value).toBeFalsy();
-    expect(pluginB.handle.mock.calls.length).toBe(0);
-
-    response.subscribe(resp => {
-      expect(resp).toEqual({
-        data: { answer: 42 },
-        status: 200,
-        statusText: 'OK',
-        headers: {}
-      });
-      done();
-    });
+    /* The second plugin should not be called as it doesn't match the condition. */
+    expect(pluginB.handle).not.toBeCalled();
+
+    const responseObserver = jest.fn();
+
+    response$.subscribe(responseObserver);
+
+    expect(responseObserver).toHaveBeenCalledTimes(1);
+    expect(responseObserver).toHaveBeenCalledWith(
+      expect.objectContaining({
+        data: {
+          answer: 42
+        }
+      })
+    );
```

Comment:
We just want to check if the plugin is skipped correctly or not.
The exchanged data content is already covered by the test above.